stripe/src/main/java/com/stripe/android/view/CardNumberEditText.java

```diff
@@ -212,4 +228,8 @@ private static int getLengthForBrand(@Card.CardBrand String cardBrand) {
     interface CardNumberCompleteListener {
         void onCardNumberComplete();
     }
+
+    interface CardBrandChangeListener {
+        void onCardBrandChanged(@NonNull @Card.CardBrand String brand);
```

Comment:
I opted for a separate listener rather than combining both into one interface in order to keep setListener methods cleaner in CardInputView